src/Interpreters/InterpreterSetQuery.h

```diff
@@ -23,7 +23,7 @@ class InterpreterSetQuery : public IInterpreter, WithMutableContext
     /** Set setting for current context (query context).
       * It is used for interpretation of SETTINGS clause in SELECT query.
       */
-    void executeForCurrentContext(bool ignore_setting_constraints = false);
```

Comment:
Is it better without default? I think it's not clear what `executeForCurrentContext(false)` means without looking into the definition (or we can add comments `executeForCurrentContext(/* ignore_setting_constraints = */ false`)